package.json

```diff
@@ -63,64 +65,64 @@
     "sassdoc": "^2.5.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.14.6",
-    "@babel/preset-env": "^7.14.5",
-    "@testing-library/dom": "^7.31.2",
-    "@testing-library/jest-dom": "^5.14.1",
-    "@yarnpkg/pnpify": "^2.4.0",
+    "@babel/core": "^7.16.7",
+    "@babel/preset-env": "^7.16.7",
+    "@testing-library/dom": "^8.11.1",
+    "@testing-library/jest-dom": "^5.16.1",
+    "@yarnpkg/sdks": "^2.6.0-rc.1",
     "accoutrement": "^3.0.1",
```

Comment:
@mirisuzanne I think upgrading Accoutrement is the remaining task here? Right now there are a lot of warnings: `Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.`